### DISCUSSION TITLE
Missing Fill method

### DISCUSSION THREAD

**Discussion by M0n7y5** [SixLabors/ImageSharp#1792] [Q&A] [answered]
Hi i am trying to follow guide from your web and i am unable to compile simple sample code due to missing `Fill` method
Is there some break api change that was not corrected in docs? 

```csharp
// draws a star with Horizontal red and blue hatching with a dash dot pattern outline.
image.Mutate( x=> x.Fill(options, brush, yourPolygon)
                   .Draw(option, pen, yourPolygon));
```

I am using Net Core 3.1

**Comment by tocsoft** [comment, marked as answer]
ensure you add the https://www.nuget.org/packages/SixLabors.ImageSharp.Drawing package (it is still pre-release)

and you have the `SixLabors.ImageSharp.Drawing.Processing` namespace included in your `using`s

**Reply by M0n7y5** [reply]
Oh its pre-release so thats why ... Anyways thanks. That should do it!
